dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/RenderFieldStrategy.java

```diff
@@ -111,11 +111,11 @@ public static Object renderFieldValue(final HttpServletRequest request,
         Try.runRunnable(()-> com.dotmarketing.util.VelocityUtil
                 .getEngine()
                 .evaluate(context, evalResult, "", fieldValueAsStr)).onFailure((error)->
-                Logger.error(RenderFieldStrategy.class, "Unable to render velocity in field: "
+                Logger.warnAndDebug(RenderFieldStrategy.class, "Unable to render velocity in field: "
                         + field.variable(), error)
-                );
```

Comment:
do we log the error in velocity?

Reply:
As warnAndDebug. Looks like this:
```
12:02:36.456  WARN  strategy.RenderFieldStrategy - Unable to render velocity in field: body
12:02:36.456  WARN  strategy.RenderFieldStrategy - Lexical error,   Encountered: "\"" (34), after : "\\" at *unset*[line 1, column 616]
12:02:36.456  WARN  strategy.RenderFieldStrategy - org.apache.velocity.runtime.RuntimeInstance.evaluate(RuntimeInstance.java:1269)
```
@wezell